tests/cpp/core/LoggingUtilityTest.cpp

```diff
@@ -0,0 +1,199 @@
+/*                                                                   */
+/*   2. Redistributions in binary form must reproduce the above      */
+/*      copyright notice, this list of conditions and the following  */
+/*      disclaimer in the documentation and/or other materials       */
+/*      provided with the distribution.                              */
+/*                                                                   */
+/*    THIS  SOFTWARE IS PROVIDED  BY THE  UNIVERSITY OF  TEXAS AT    */
+/*    AUSTIN  ``AS IS''  AND ANY  EXPRESS OR  IMPLIED WARRANTIES,    */
+/*    INCLUDING, BUT  NOT LIMITED  TO, THE IMPLIED  WARRANTIES OF    */
+/*    MERCHANTABILITY  AND FITNESS FOR  A PARTICULAR  PURPOSE ARE    */
+/*    DISCLAIMED.  IN  NO EVENT SHALL THE UNIVERSITY  OF TEXAS AT    */
+/*    AUSTIN OR CONTRIBUTORS BE  LIABLE FOR ANY DIRECT, INDIRECT,    */
+/*    INCIDENTAL,  SPECIAL, EXEMPLARY,  OR  CONSEQUENTIAL DAMAGES    */
+/*    (INCLUDING, BUT  NOT LIMITED TO,  PROCUREMENT OF SUBSTITUTE    */
+/*    GOODS  OR  SERVICES; LOSS  OF  USE,  DATA,  OR PROFITS;  OR    */
+/*    BUSINESS INTERRUPTION) HOWEVER CAUSED  AND ON ANY THEORY OF    */
+/*    LIABILITY, WHETHER  IN CONTRACT, STRICT  LIABILITY, OR TORT    */
+/*    (INCLUDING NEGLIGENCE OR OTHERWISE)  ARISING IN ANY WAY OUT    */
+/*    OF  THE  USE OF  THIS  SOFTWARE,  EVEN  IF ADVISED  OF  THE    */
+/*    POSSIBILITY OF SUCH DAMAGE.                                    */
+/*                                                                   */
+/* The views and conclusions contained in the software and           */
+/* documentation are those of the authors and should not be          */
+/* interpreted as representing official policies, either expressed   */
+/* or implied, of Ecole polytechnique federale de Lausanne.          */
+/*********************************************************************/
+
+#define BOOST_TEST_MODULE LoggingUtilityTest
+
+#include <boost/make_shared.hpp>
+#include <boost/regex.hpp>
+#include <boost/test/unit_test.hpp>
+namespace ut = boost::unit_test;
+
+#include "DisplayGroup.h"
+#include "DummyContent.h"
+#include "LoggingUtility.h"
+#include "rest/RestLogger.h"
+
+#include <QObject>
+
+namespace
+{
+const QSize wallSize( 1000, 1000 );
+const std::string regexJson{
+R"(\{
+    "event": \{
+        "last_event": "contentWindowAdded",
+        "last_event_date": "\d{4}-\d{2}-\d{2}\u\d{2}:\d{2}:\d{2}.\d{6}",
+        "recent_events": 2
+    \},
+    "window": \{
+        "counter": 2,
+        "date_set": "\d{4}-\d{2}-\d{2}\u\d{2}:\d{2}:\d{2}.\d{6}",
+        "total_counter": 2
+    \}
+\}
+)"
+};
+const std::string defaultJson{
+R"({
+    "event": {
+        "last_event": "",
+        "last_event_date": "",
+        "recent_events": 0
+    },
+    "window": {
+        "counter": 0,
+        "date_set": "",
+        "total_counter": 0
+    }
+}
+)"
+};
+}
+
+BOOST_AUTO_TEST_CASE( testAccumulatedWindowCount )
```

Comment:
The json uses "total count" instead of "accumulated count", can we choose one or the other?